File: src/main/java/com/extendedae_plus/mixin/bridge/BridgeProviderList.java
package com.extendedae_plus.mixin.bridge;

import appeng.api.implementations.blockentities.PatternContainerGroup;
import appeng.helpers.patternprovider.PatternContainer;

import java.util.List;
import java.util.Map;

public interface BridgeProviderList {
    Map<PatternContainerGroup, List<PatternContainer>> eaep$getProviderList();
}

File: src/main/kotlin/com/extendedae_plus/mixin/helper/BridgeProviderList.kt
package com.extendedae_plus.mixin.helper

import appeng.api.implementations.blockentities.PatternContainerGroup
import appeng.helpers.patternprovider.PatternContainer

interface BridgeProviderList {
    fun `eaep$getProviderList`(): Map<PatternContainerGroup, List<PatternContainer>>
}
